weekly/2024/week-27.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Week 27 in Packit
+date: 2024-07-08
+authors: mmassari
+tags:
+  - 2024-July
+  - 2024
+  - July
+---
+
+## Week 27 (July 2nd – July 8th)
+
+- Packit now updates its own, not yet merged, pull requests instead of creating new ones for new releases. ([packit#2204](https://github.com/packit/packit/pull/2204))
```

Comment:
```suggestion
- Packit now updates its own, not yet merged, pull requests instead of creating new ones when syncing new releases downstream. ([packit#2204](https://github.com/packit/packit/pull/2204))
```